cvmfs/prng.h

```diff
@@ -49,6 +50,30 @@ class Prng {
     return (uint32_t)scaled_val % boundary;
   }
 
+  /**
+   * Returns random double in range [0, 1]
+   */
+  double NextDouble() {
+    state_ = a*state_ + c;
+    double unit_val = static_cast<double>(state_) /
+      static_cast<double>(18446744073709551616.0);
+    return unit_val;
+  }
+  /**
+   * Returns normally distributed random numbers
+   * with mean 0 and variance 1 using the
+   * Box-Muller transform algorithm
+   */
+  double NextNormal() {
```

Comment:
I think there should be a unit test for this function.  It could generate, say, 100000 random numbers and check with a very generous margin that mean and variance are indeed 0 and 1.